schedule/kernel/autoyast_create_hdd_multipath_lvm_textmode.yaml

```diff
@@ -0,0 +1,25 @@
+name: multipath
+
+description:
+    Prepare image with multipath and lvm enabled using autoyast installation for kernel testing.
+
+vars:
+    DESKTOP: textmode
+    MULTIPATH: 1
+    LVM: 1
+    HDDMODEL: scsi-hd
+    SEPARATE_HOME: 0
+    INSTALLONLY: 1
+    AUTOYAST: autoyast_sle15/create_hdd/create_hdd_multipath_lvm_textmode_%ARCH%.xml
```

Comment:
i am not sure how you plan to setup the test. in case this is a new testsuite that wouldnt make sense here. in additional mixing variables doesnt look good. for instance `HDDMODEL` should come from machine variables. Can you shed some light why do we need the `vars` section in the yaml?

Reply:
that is correct, `HDDMODEL` needs to be defined at test suite level to have impact, before is vm created (it's late in yaml), but it doesn't do any harm here.    Ir was always like this and we didn't consider to create new machine definition for it, so it is fine, like it is now.

Reply:
Thank you @b10n1k and @czerw for the explanation. I think I just based this on the existing yaml - `create_hdd_multipath_lvm_extmode.yaml`
Please confirm if I should remove the vars other than AUTOYAST.

Reply:
i am not sure what @czerw says.
However my emphasis is what reasoning the use of the variables in the yaml here. i still dont understand if we have a good explanation having the variables here. At least all of them. there are different approaches to schedule the job and i guess the main difference is the AUTOYAST. That can be defined here (depends!). But the rest of the variables are already defined in the `create_hdd_multipath_lvm_textmode`.

Here is some of the flows:
1. create a new testsuite
2. override the `create_hdd_multipath_lvm_textmode` in the jobgroup

if you do the 1 all those variables need to be in testsuite. but you kinda duplicate entries in the db
if you go with 2 all the variables except AUTOYAST do not need to be here as you can reuse the `create_hdd_multipath_lvm_textmode`. (or set them there but this is just matter of personal taste). 

Do you see my objection or do we need to have a chat?

Reply:
I think we are following the option 1 of creating new testsuites for autoyast based installation. And I think we can remove the existing testsuites once we have moved to the new ones completely. @czerw, please correct me if I am wrong.
So all these variables except AUTOYAST should be moved to new testsuite settings in osd?

Reply:
@b10n1k  i don't understand you too :)

Reply:
As per the current situation we have `autoyast_create_hdd_multipath_lvm_encrypt_textmode` `autoyast_create_hdd_multipath_lvm_textmode` and `autoyast_create_hdd_multipath_textmode` (new test suites). 

What you can do is to
1. either remove the `DESKTOP=textmode` from there to not get confused.
2. move all the variables into the new test suites

Reply:
@b10n1k , I removed the vars from the yaml in updated commit and added them in the test suite in osd.

Reply:
@Avinesh thanks. and sorry for making this that long :)

Reply:
Thank you @b10n1k for all the suggestions.